services/storage/src/simcore_service_storage/utils_handlers.py

```diff
@@ -36,12 +40,15 @@ async def dsm_exception_handler(
     except ValidationError as err:
         raise web.HTTPUnprocessableEntity(reason=f"{err}") from err
     except DBAPIError as err:
+        _logger.exception("unexpected error")
```

Comment:
I am aware that this prints a stack en extra info on the exception but can you perhaps add some meaningful generic mesage? e.g. 
- `Unexpected error in the database. Responding with 503` for this
- `Unexpected error in S3. Responding with 503`` for the next and 
- `Unepected timeout in data-core service. Responding with 504`

Reply:
I agree,
proposals:
```Unexpected error while accessing DB:```
```Unexpected error while accessing S3:```
```Unexpected error while access Datcore-Adapter:```

Reply:
I agree,
proposals:
```Unexpected error while accessing DB:```
```Unexpected error while accessing S3:```
```Unexpected error while access Datcore-Adapter:```